### PR TITLE
Changes to open existing map to remove references to portal

### PR DESCRIPTION
**Pull request by macca400**
-Changes to open existing map sample to remove search and portal functionality. 
-Kept hard coding to a minimum. 
-Changed sample class name to follow naming convention followed by other .Net samples

### CODE REVIEW

**Comment by macca400** [comment]
@anttikajanus can you review please?

File: src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/OpenExistingMap/OpenExistingMap.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        private string title1 = "Housing with Mortgages";
+        private string title2 = "USA Tapestry Segmentation";
+        private string title3 = "Geology of United States";
+
+        //Construct Load Map sample control
+        public OpenExistingMap()
+        {
+            InitializeComponent();
+
+            Loaded += LoadMap_Loaded;
```
Comment (anttikajanus) [inline src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/OpenExistingMap/OpenExistingMap.xaml.cs]: let's follow naming conventions with usercontrol based events too so OnLoaded

File: src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/OpenExistingMap/OpenExistingMap.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        }
+
+        // Loads UI elements and an initial Map
+        private async void LoadMap_Loaded(object sender, RoutedEventArgs e)
+        {
+            ICollection<KeyValuePair<String, String>> comboBoxContent =
+            new Dictionary<String, String>()
```
Comment (anttikajanus) [inline src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/OpenExistingMap/OpenExistingMap.xaml.cs]: Intent one more

File: src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/OpenExistingMap/OpenExistingMap.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        {
+            InitializeComponent();
+
+            Loaded += LoadMap_Loaded;
+        }
+
+        // Loads UI elements and an initial Map
+        private async void LoadMap_Loaded(object sender, RoutedEventArgs e)
+        {
+            ICollection<KeyValuePair<String, String>> comboBoxContent =
```
Comment (anttikajanus) [inline src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/OpenExistingMap/OpenExistingMap.xaml.cs]: Comment what you are doing

File: src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/OpenExistingMap/OpenExistingMap.xaml.cs [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+        // Loads the given map
+        private async Task LoadMapAsync(string mapUrl)
+        {
+            try
+            {
+                progress.Visibility = Visibility.Visible;
+
+                //Initialize map from a portal item URI and load map into MapView 
+                var map = new Map(new Uri(mapUrl));
+                await map.LoadAsync();
```
Comment (anttikajanus) [inline src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/OpenExistingMap/OpenExistingMap.xaml.cs]: Is this needed?

File: src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/OpenExistingMap/OpenExistingMap.xaml.cs [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+            }
+            catch (Exception ex)
+            {
+                var errorMessage = "Map cannot be loaded. " + ex.Message;
+                MessageBox.Show(errorMessage, "Sample error");
+            }
+        }
+
+        // Loads a webmap on load button click
+        private async void OnLoadButton_Click(object sender, RoutedEventArgs e)
```
Comment (anttikajanus) [inline src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/OpenExistingMap/OpenExistingMap.xaml.cs]: naming conventions

File: src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/OpenExistingMap/OpenExistingMap.xaml.cs [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+using System.Threading.Tasks;
+using System.Windows;
+using System.Collections.Generic;
+
+
+namespace ArcGISRuntime.Desktop.Samples.OpenExistingMap
+{
+    public partial class OpenExistingMap
+    {
+        private string itemURL1 = "http://www.arcgis.com/home/item.html?id=2d6fa24b357d427f9c737774e7b0f977";
```
Comment (anttikajanus) [inline src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/OpenExistingMap/OpenExistingMap.xaml.cs]: comment what these are

**Comment by macca400** [comment]
Closing and using newer branch for merge to avoid Desktop/desktop windows folder issue. new PR https://github.com/Esri/arcgis-runtime-samples-dotnet/pull/131

**Pull request closed without merging**